index.html

```diff
@@ -7,9 +7,9 @@
 <h2>Latest releases of the FHIR tools:</h2>

```

Comment:
We can get rid of this file now

Reply:
So, the `index.html` is being updated with `tools.json` at https://github.com/FHIR/latest-ig-publisher. Then, after fetching upstream and rebasing upstream/master, we should manually remove `index.html` before pushing new commits. Do we want that?

Reply:
Oh I see, I didn't know it was still a part of the original repo. Let's leave it in